extensions/aws/processors/PutS3Object.h

```diff
@@ -26,6 +26,8 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <chrono>
+#include <atomic>
```

Comment:
I think `<atomic>` is no longer needed, unless I'm missing something

Reply:
Removed in 59dd271565f5ea1d64cce787dc29305fbe8667e3